server/src/test/scala/cromwell/subworkflowstore/SubWorkflowStoreSpec.scala

```diff
@@ -28,7 +28,7 @@ object SubWorkflowStoreSpec {
   val EmptyExpression = WdlExpression.fromString(""" "" """)
 }
 
-class SubWorkflowStoreSpec extends CromwellTestKitWordSpec with CoordinatedWorkflowStoreBuilder with Matchers with Mockito {
+class SubWorkflowStoreSpec extends CromwellTestKitWordSpec with CoordinatedWorkflowStoreActorBuilder with Matchers with Mockito {
```

Comment:
Mixed in `CromwellTimeoutSpec` to all modified `*Spec` files with the exception of `WorkflowStoreCoordinatedAccessActorSpec.scala`.

It inherits from `akka.testkit.TestKit` instead of `org.scalatest.TestSuite` and is therefore not applicable.